site/content/cluster/create-database.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-curl -X POST http://localhost:7201/api/v1/database/create -d '{
+curl -X POST {{% apiendpoint %}}database/create -d '{
```

Comment:
Looks like replacements inside the code block don't always work properly ([link](https://deploy-preview-3281--m3db.netlify.app/docs/cluster/kubernetes_cluster/#create-a-placement-and-namespace)):
![image](https://user-images.githubusercontent.com/8646655/110470764-ee5dfa80-80e3-11eb-827f-e7929b8e52e2.png)

Reply:
Hmm, that's odd, I tested that locally 🤔

Reply:
And it was working before…